subscriptions-api.md

```diff
@@ -0,0 +1,613 @@
+# CNCF CloudEvents – Subscriptions API
+
+## Abstract
+
+This specification defines mechanisms, including an API definition, for CNCF
+CloudEvents event consumers to subscribe to events originating from event
+producers on behalf of event sources. The software entity handling these
+subscriptions and responsible for distributing events is abstractly referred to
+as a "subscription manager".
+
+## Status of this document
+
+This document is a working draft.
+
+## 1. Introduction
+
+A subscription manager responsible for a specific set of events can reside
+immediately at or near the event producer or it can reside in some middleware
+infrastructure. This document does not formalize the relationship between the
+original event producer and the subscription manager, and how the subscription
+manager obtains the events it distributes to subscribers.
+
+This document also does not formalize the relationship between a specific set of
+events and the subscription manager. An event producer that produces multiple
+different types of events can offer one subscription manager for all events or a
+subscription manager for each type of event, or any other combination. An event
+producer may also offer up the same set of events for subscription at multiple
+concurrent subscription managers. Such subscription offers are published in a
```

Comment:
Does the "Such" refer to all of the cases listed in this paragraph or just to the last case mentioned? I think the intent was to cover all of them, if so, perhaps: Regardless of relationship between the event producer(s) and the subscription manager(s), the advertisement of the subscription offers are published in a CloudEvents Discovery service

Maybe?